src/osx/core/uilocale.mm

```diff
@@ -150,6 +150,8 @@ int CompareStrings(const wxString& lhs, const wxString& rhs,
 wxUILocaleImplCF::GetLocalizedName(wxLocaleName name, wxLocaleForm form) const
 {
     NSString* str = nullptr;
+    NSLocale *enLocale = [[NSLocale alloc] initWithLocaleIdentifier:@"en_US"];
+    [enLocale autorelease];
```

Comment:
Not sure if it's really an expensive operation, but we could avoid creating this locale until it's really needed relatively easily by just doing
```suggestion
    auto const getEnLocale = []() {
        NSLocale *enLocale = [[NSLocale alloc] initWithLocaleIdentifier:@"en_US"];
        [enLocale autorelease];
        return enLocale;
    };
```
and then using `getEnLocale()` below. Any reason not to do it like this?